server/routes/journeys/develop-an-initial-plan/next-review-date/tests.cy.ts

```diff
@@ -65,11 +65,8 @@ context('Make a Referral Journey', () => {
       .clear()
       .type(`${today.getDate()}/${today.getMonth() + 1}/${today.getFullYear()}`)
     cy.findByRole('button', { name: /choose date/i }).click()
-    const currentMonthLong = new Intl.DateTimeFormat('en-GB', { month: 'long' }).format(today)
-    cy.findAllByRole('button', { name: new RegExp(`Excluded date.*${currentMonthLong}.*`, 'i'), hidden: true }).should(
-      'have.length',
-      1,
-    )
+    cy.findByRole('button', { name: /Previous month/i }).click()
+    cy.findAllByRole('button', { name: /Excluded date.*/i, hidden: true }).should('have.length.greaterThan', 27)
```

Comment:
Won't this fail when we're nearer the end of the month?

Reply:
It's always going back to the previous month, so they should always be all excluded. It's not a great test for "in the past" but it will at least catch whether its been specified or not

Reply:
For some reason I thought it shows days in previous month's like some other date pickers do, glad it doesn't! Updated to check for "today - 1" days in the current month and making sure last month is excluded (for when the test runs on the first day of the month)